### PR TITLE
Fixes custom material examine runtime

### PR DESCRIPTION
**Pull request by ShizCalev**
old
image
fixed
image

:cl: ShizCalev
fix: Fixed examine messages stating what materials something is made out of not working properly.
/:cl:

### CODE REVIEW

File: code/game/atoms.dm [hunk 1]
```diff
@@ -694,9 +694,11 @@
 
 	if(custom_materials)
 		var/list/materials_list = list()
-		for(var/datum/material/current_material as anything in custom_materials)
+		for(var/custom_material as anything in custom_materials)
+			var/datum/material/current_material = GET_MATERIAL_REF(custom_material)
```
Comment (JohnFulpWillard) [inline code/game/atoms.dm:698]: You don't need to ``as anything`` with no path to check

File: code/game/atoms.dm [hunk 2]
```diff
@@ -694,9 +694,11 @@
 
 	if(custom_materials)
 		var/list/materials_list = list()
-		for(var/datum/material/current_material as anything in custom_materials)
+		for(var/custom_material as anything in custom_materials)
```
Comment (ShizCalev) [inline code/game/atoms.dm:697]:
```suggestion
		for(var/custom_material in custom_materials)
```

File: code/game/objects/structures/tables_racks.dm [hunk 3]
```diff
@@ -334,7 +334,8 @@
 /obj/structure/table/greyscale/set_custom_materials(list/materials, multiplier)
 	. = ..()
 	var/list/materials_list = list()
-	for(var/datum/material/current_material as anything in custom_materials)
+	for(var/custom_material as anything in custom_materials)
```
Comment (ShizCalev) [inline code/game/objects/structures/tables_racks.dm:337]:
```suggestion
	for(var/custom_material in custom_materials)
```

File: code/game/atoms.dm (hunk 1 above)
Comment (ShizCalev) [reply code/game/atoms.dm]: Just habit haha

**Comment by JohnFulpWillard** [comment]
damnation daniel

**Pull request merged**